### PR TITLE
[MAIN] Removal of MedStickers

### PR DESCRIPTION
**Pull request by mmsarquis**

### CODE REVIEW

File: Example/Krypt.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings [hunk 1]
```diff
@@ -4,5 +4,7 @@
 <dict>
 	<key>BuildSystemType</key>
 	<string>Original</string>
+	<key>PreviewsEnabled</key>
```
Comment (hngry) [inline Example/Krypt.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings]: Is this for SwiftUI?

**Review by markotect — COMMENTED** [review]
🧹

File: Example/Krypt.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings (hunk 1 above)
Comment (marcelosarquis) [reply Example/Krypt.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings]: Yes. It was added by Xcode automatically. Should I remove it?

> Xcode 11 displays previews of your user interface right in the editor, streamlining the edit-debug-run cycle into a seamless workflow. Optimize the structure of your SwiftUI app for previews, and how to add preview support to your existing views and view controllers.

Comment (hngry) [reply Example/Krypt.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings]: I'm just wondering why it's `false`

**Review by markotect — APPROVED** [review]

**Pull request merged**